.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
     - MPLBACKEND=agg
     - AODNFETCHER_URL="git+https://github.com/aodn/python-aodnfetcher.git@master"
     - AODNCORE_URL="jenkins://imos-binary/aodncore_prod?pattern=^.*\.whl$"
+    - AODNUTILS_URL="jenkins://imos-binary/ncwriter?pattern=^.*\.whl$"
```

Comment:
Maybe just change it to "NCWRITER_URL" so it matches the name of the package for clarity?

Reply:
done !